### PR TITLE
Small Bug Fix and CodeSandbox v5/v6 seperation

### PR DESCRIPTION
**Pull request by pmaier983**
- make it possible to click `copy` to copy TS code.
- clearly separate v5 and v6 sandbox's
- fix and update the custom validation resolver codesandbox's and displayed code

### CODE REVIEW

File: src/components/codeExamples/validationResolver.ts [hunk 1]
```diff
@@ -1,40 +1,52 @@
...
-    errors: error
-      ? error.details.reduce((previous, currentError) => {
-          return {
-            ...previous,
-            [currentError.path[0]]: currentError
-          };
-        }, {})
-      : {}
+  const onSubmit = data => {
+    alert(JSON.stringify(data));
```
Comment (bluebill1049) [inline src/components/codeExamples/validationResolver.ts:36]: can we please use `console.log` for consistency?

File: src/components/codeExamples/validationResolver.ts [hunk 2]
```diff
@@ -1,40 +1,52 @@
...
-    context: { test: "test" }
-  });
 
   return (
-    <form onSubmit={handleSubmit(d => console.log(d))}>
-      <input type="text" name="username" ref={register} />
-      <input type="submit" />
-    </form>
+    <div className="App">
+      <h1>validationResolver</h1>
```
Comment (bluebill1049) [inline src/components/codeExamples/validationResolver.ts:41]: I think it's just `resolver` now

**Comment by bluebill1049** [comment]
Thanks a lot for the PR @pmaier983 just some minor feedback.

File: src/components/codeExamples/validationResolver.ts (hunk 1 above)
Comment (pmaier983) [reply src/components/codeExamples/validationResolver.ts:36]: Keep forgetting to change this after copy-paste from SandBox. Will Do.

**Pull request merged**